src/xunit.performance.api/Profilers/Etw/EtwAddressRange.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    internal class EtwAddressRange
```

Comment:
If the underlying types make sense, then we could use it on non-Windows (I do not know that yet). This is more of an abstraction of the tracing library. Would it make sense to make it non-Etw?

---
In reply to: [142810045](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r142810045) [](ancestors = 142810045)

Reply:
Per our conversation, I think it's reasonable to keep these as ETW if they are internal and we can make a call.  For public types, we should think harder because we will support non-ETW profilers.

---
In reply to: [143057784](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143057784) [](ancestors = 143057784,142810045)